python/mkdocs/docs/api.md

```diff
@@ -150,32 +150,25 @@ catalog.create_table(
 
 Add new columns through the `Transaction` or `UpdateSchema` API:
 
-Use the Transaction API:
-
 ```python
-with table.transaction() as transaction:
-    transaction.update_schema().add_column("x", IntegerType(), "doc").commit()
+with table.update_schema() as update:
+    update.add_column("x", IntegerType(), "doc")
 ```
 
-Or, without a context manager:
+Or, without a context manager by calling the `.commit()` explicitly:
 
 ```python
-transaction = table.transaction()
-transaction.update_schema().add_column("x", IntegerType(), "doc").commit()
-transaction.commit_transaction()
+table.update_schema().add_column("x", IntegerType(), "doc").commit()
 ```
 
-Or, use the UpdateSchema API directly:
+Use the Transaction API:
```

Comment:
What about "Alternatively, use the transaction API to combine changes from multiple operations"? Otherwise it isn't clear why you'd use a transaction.

Reply:
I tried to indicate that by also setting a property in the same transaction, but I agree that it is better to explicitly state it. Thanks!

Reply:
```suggestion
Alternatively, use the transaction API to combine changes from multiple operations:
```